### PR TITLE
feat: Add Lambda Authorizer

### PR DESCRIPTION
**Pull request by PavelZaletski**
**Completed tasks:**

1 - authorization-service is added to the repo, has correct basicAuthorizer lambda and correct serverless.yaml file
3 - import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda. Request to the importProductsFile lambda should work only with correct authorization_token being decoded and checked by basicAuthorizer lambda. Response should be in 403 HTTP status if access is denied for this user (invalid authorization_token) and in 401 HTTP status if Authorization header is not provided.
5 - update client application to send Authorization: Basic authorization_token header on import. Client should get authorization_token value from browser localStorage https://developer.mozilla.org/ru/docs/Web/API/Window/localStorage authorization_token = localStorage.getItem('authorization_token')

**Additional (optional) tasks**
+1 - Client application should display alerts for the responses in 401 and 403 HTTP statuses. This behavior should be added to the nodejs-aws-fe-main/src/index.tsx file

**Frontend part:**
CloudFront - https://dsb3s2ts35uzs.cloudfront.net/admin/products
authorization_token=cGF2ZWx6YWxldHNraTpURVNUX1BBU1NXT1JE
PR - https://github.com/PavelZaletski/nodejs-aws-fe/pull/4

Upload file example:

title,description,price,img,count
Samsung%20QN75Q8DT%20Test4,Short%20Product%20Description10,1500,https%3A%2F%2Fcrdms.images.consumerreports.org%2Ff_auto%2Cw_300%2Fprod%2Fproducts%2Fcr%2Fmodels%2F401740-60-inch-and-larger-tvs-samsung-qn75q8dt-10014881.png,36

### CODE REVIEW

**Comment by VestryOd** [comment]
image
Doesn't work properly.

**Comment by PavelZaletski** [comment]
Sorry, forgot to deploy the latest code changes. Now it should work.
Screen Shot 2020-12-01 at 12 46 07 PM

File: authorization-service/serverless.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    basicAuthorizer: {
+      handler: 'handler.basicAuthorizer',
+      events: [
+        {
+          http: {
+            method: 'get',
+            path: 'import',
+          }
+        }
+      ]
```
Comment (sergeybolzan) [inline authorization-service/serverless.ts]: not needed

Comment (PavelZaletski) [reply authorization-service/serverless.ts]: Removed.

**Pull request merged**